### ISSUE TITLE
Fuzzy match in keybinding search

### ISSUE THREAD

**Issue by octref** [atom/atom#11536]
### Description

`pane:show` would list a bunch of keybindings.

![image](https://cloud.githubusercontent.com/assets/4033249/14660893/47c1f5b4-0677-11e6-82f3-1016278395ba.png)

`pane show` shows none.

![image](https://cloud.githubusercontent.com/assets/4033249/14660899/540c2358-0677-11e6-8d1d-9d449fc9556f.png)

### Steps to Reproduce

Open Keybindings, type in `pane show`.

**Expected behavior:** Shows all keybindings that have a command including the word `pane` and `show`.
Basically a word-wise fuzzy match.

**Actual behavior:** Search returns no result.

### Versions

OS X 10.11.4
Atom 1.7.2

I can work on this and make a PR.

**Comment by 50Wliu** [comment]
Hi @octref, can you please move this to atom/settings-view?  Thanks!

(And fuzzy-searching would be awesome - feel free to make a PR!)
